source/cpu/cpu.go

```diff
@@ -221,6 +232,15 @@ func (s *cpuSource) GetFeatures() *feature.DomainFeatures {
 	return s.features
 }
 
+func getCPUModel() map[string]string {
+	cpuModelInfo := make(map[string]string)
+	cpuModelInfo["vendorID"] = cpuid.CPU.VendorID.String()
+	cpuModelInfo["family"] = strconv.Itoa(cpuid.CPU.Family)
+	cpuModelInfo["ID"] = strconv.Itoa(cpuid.CPU.Model)
```

Comment:
nit: I wouldn't probably use `camelCase` in the labels (and not `CamelCase` either). If something, I think, the prevailing style so far is `snake_case` so I'd suggest to use it. WDYT?
 
```suggestion
	cpuModelInfo["vendor_id"] = cpuid.CPU.VendorID.String()
	cpuModelInfo["family"] = strconv.Itoa(cpuid.CPU.Family)
	cpuModelInfo["id"] = strconv.Itoa(cpuid.CPU.Model)
```